8/README.MD

```diff
@@ -461,19 +454,13 @@ This will print out "Input a number" in the console, wait for the user to input
 
 **Classes** are what define what values an object will hold and what methods it will have associated with it. Think of a **class** as the recipe you use to make an **object**.
 
-Classes can have **private** and **public** methods.
```

Comment:
Misleading to say Python has public and private methods. A `_` prefix is just a convention to signal to callers that they shouldn't call a method. And `__` just induces mangling, which allows subclasses to call ancestors' versions of methods. But all such methods can still be called publicly, even if ill-advised.